references/obj_detection/train_pytorch.py

```diff
@@ -24,17 +24,18 @@
 from doctr.datasets import DocArtefacts
 from doctr.models import obj_detection
 from doctr.utils import DetectionMetric
+from references.obj_detection.utils import plot_samples
 from utils import plot_recorder
 
 
 def record_lr(
-    model: torch.nn.Module,
-    train_loader: DataLoader,
-    optimizer,
-    start_lr: float = 1e-7,
-    end_lr: float = 1,
-    num_it: int = 100,
-    amp: bool = False,
+        model: torch.nn.Module,
+        train_loader: DataLoader,
+        optimizer,
+        start_lr: float = 1e-7,
+        end_lr: float = 1,
+        num_it: int = 100,
+        amp: bool = False,
```

Comment:
? I think your IDE did some automatic formatting :sweat_smile:

Reply:
Yup, It was responsible :) I will take care of it from next time.